code/__defines/planets.dm

```diff
@@ -11,6 +11,7 @@
 #define WEATHER_BLOOD_MOON			"blood moon" // For admin fun or cult later on.
 #define WEATHER_EMBERFALL			"emberfall" // More adminbuse, from TG. Harmless.
 #define WEATHER_ASH_STORM			"ash storm" // Ripped from TG, like the above. Less harmless.
+#define WEATHER_ASH_STORM_SAFE		"ash storm (Safe)" //Safe version of the ash storm. Dimmer.
```

Comment:
considering that name is used ingame for IC stuff, I'd suggest a usage of name like "light ash storm" or something among those lines

Reply:
Ah, makes sense. I'll go ahead and change that real quick